integration-tests/return-response/src/main/resources/application.properties

```diff
@@ -1,4 +1,24 @@
+quarkus.openapi-generator.codegen.spec.return_response_false_void_simple_openapi_yaml.base-package = org.acme.openapi
+quarkus.openapi-generator.codegen.spec.return_response_true_string_simple_openapi_yaml.base-package = org.acme.openapi
+quarkus.openapi-generator.codegen.spec.return_response_true_void_simple_openapi_yaml.base-package = org.acme.openapi
 
-quarkus.openapi-generator.codegen.spec.response_simple_openapi_yaml.base-package=org.acme.openapi
-quarkus.openapi-generator.codegen.spec.response_simple_openapi_yaml.return-response=true
\ No newline at end of file
+quarkus.openapi-generator.codegen.spec.return_response_false_string_simple_openapi_yaml.return-response = false
+quarkus.openapi-generator.codegen.spec.return_response_false_void_simple_openapi_yaml.return-response = false
+quarkus.openapi-generator.codegen.spec.return_response_true_string_simple_openapi_yaml.return-response = true
+quarkus.openapi-generator.codegen.spec.return_response_true_void_simple_openapi_yaml.return-response = true
+
+quarkus.openapi-generator.codegen.spec.mutiny_return_response_false_string_simple_openapi_yaml.base-package = ${quarkus.openapi-generator.codegen.spec.return_response_false_string_simple_openapi_yaml.base-package}
+quarkus.openapi-generator.codegen.spec.mutiny_return_response_false_void_simple_openapi_yaml.base-package = ${quarkus.openapi-generator.codegen.spec.return_response_false_void_simple_openapi_yaml.base-package}
+quarkus.openapi-generator.codegen.spec.mutiny_return_response_true_string_simple_openapi_yaml.base-package = ${quarkus.openapi-generator.codegen.spec.return_response_true_string_simple_openapi_yaml.base-package}
+quarkus.openapi-generator.codegen.spec.mutiny_return_response_true_void_simple_openapi_yaml.base-package = ${quarkus.openapi-generator.codegen.spec.return_response_true_void_simple_openapi_yaml.base-package}
+
+quarkus.openapi-generator.codegen.spec.mutiny_return_response_false_string_simple_openapi_yaml.return-response = ${quarkus.openapi-generator.codegen.spec.return_response_false_string_simple_openapi_yaml.return-response}
+quarkus.openapi-generator.codegen.spec.mutiny_return_response_false_void_simple_openapi_yaml.return-response = ${quarkus.openapi-generator.codegen.spec.return_response_false_void_simple_openapi_yaml.return-response}
+quarkus.openapi-generator.codegen.spec.mutiny_return_response_true_string_simple_openapi_yaml.return-response = ${quarkus.openapi-generator.codegen.spec.return_response_true_string_simple_openapi_yaml.return-response}
+quarkus.openapi-generator.codegen.spec.mutiny_return_response_true_void_simple_openapi_yaml.return-response = ${quarkus.openapi-generator.codegen.spec.return_response_true_void_simple_openapi_yaml.return-response}
+
+quarkus.openapi-generator.codegen.spec.mutiny_return_response_false_string_simple_openapi_yaml.mutiny = true
+quarkus.openapi-generator.codegen.spec.mutiny_return_response_false_void_simple_openapi_yaml.mutiny = true
```

Comment:
```suggestion
quarkus.openapi-generator.codegen.spec.mutiny_return_response_false_string_simple_openapi_yaml.mutiny = false
quarkus.openapi-generator.codegen.spec.mutiny_return_response_false_void_simple_openapi_yaml.mutiny = false
```

Reply:
I mean the file is not coherent with the property value.

quarkus.openapi-generator.codegen.spec.return_response_**_true_**_string_simple_openapi_yaml.mutiny = **_false_**

Reply:
I am sorry, I again may be confused after today's work, but I don't think that `*.mutiny` has should mirror the value of `return-response`?

I basically copied the 4 existing specs and tests for the classic cases, where there are the 2 cases you added (`string` / `void`) for both options of `return-response`

- return_response_false_string_simple_openapi_yaml
- return_response_false_void_simple_openapi_yaml
- return_response_true_string_simple_openapi_yaml
- return_response_true_void_simple_openapi_yaml

and gave them reactive twins, with again the same 2 cases for `return-response`

- **mutiny**_return_response_false_string_simple_openapi_yaml
- **mutiny**_return_response_false_void_simple_openapi_yaml
- **mutiny**_return_response_true_string_simple_openapi_yaml
- **mutiny**_return_response_true_void_simple_openapi_yaml

The `*.base-package` and `*.return-response` values for each of the reactive tests mirror the values of the corresponding classic equivalent test, while the values for the `*.mutiny` flag as well as the assertions in the unit tests for `Uni` differ for these test.

Reply:
Oh, I just noticed these properties are to enable/disable mutiny 🤦🏽‍♂️
I was thinking they were enabling/disabling return-response. So it wasn't making any sense to me:
return_response_**_false_**_string_simple_openapi_yaml.return-response = **_true_**
Please forget all I said about this. Sorry for the confusion.

Reply:
Nevermind, thanks a lot for the review!